framework/core/src/User/SessionServiceProvider.php

```diff
@@ -20,12 +22,42 @@ class SessionServiceProvider extends AbstractServiceProvider
      */
     public function register()
     {
-        $this->container->singleton('session.handler', function ($container) {
-            return new FileSessionHandler(
-                $container['files'],
-                $container['config']['session.files'],
-                $container['config']['session.lifetime']
-            );
+        $this->container->singleton('flarum.session.drivers', function () {
+            return [];
+        });
+
+        $this->container->singleton('session', function (Container $container) {
```

Comment:
Why `session` and not `session_manager`?

Reply:
Laravel's binding is called `session` and our codebase seems consistent in using laravel binding names when it makes sense, like the filesystem ones.